### ISSUE TITLE
Angular 15 new project node_modules bug

### ISSUE THREAD

**Issue by jeremyspatrick** [single-spa/single-spa-angular#467]
###  Issue

ng serve has a compilation error in the node_modules folder. The file single-spa-angular.d.ts does not exist but singleSpaAngular is available in index.d.ts

<img width="1046" alt="image" src="https://user-images.githubusercontent.com/111452118/211105358-41a058ec-6dfe-4f93-ae18-7e77accee3af.png">

<img width="541" alt="image" src="https://user-images.githubusercontent.com/111452118/211105414-62a619e7-54e7-4ea2-916d-2a851141a1d9.png">

<img width="207" alt="image" src="https://user-images.githubusercontent.com/111452118/211105937-7a163e66-150e-4de1-8aed-0a3736790d0f.png">

<img width="830" alt="image" src="https://user-images.githubusercontent.com/111452118/211106157-68a06a14-71db-49ca-8b99-e6b4e22f0a7a.png">

### Steps to Reproduce

Pushed this issue to a branch called angular15

This project was created using create-single-spa with single-spa-angular@8

Steps to reproduce

``` 
git clone https://github.com/jeremyspatrick/single-spa-angular14-routing-bug
cd single-spa-angular14-routing-bug
git checkout angular15
npm i
npm start
```

### Workaround/Hack fix

I'm able to work around the issue by modifying the node_modules package

<img width="587" alt="image" src="https://user-images.githubusercontent.com/111452118/211105544-8b9d97cf-ca7c-4696-98c4-1141bcceddae.png">

### Expected Behavior

ng serve should compile and start without modifying node_modules

**Comment by arturovt** [comment]
This is an issue of the `@nrwl/angular:package` builder. I've created an issue in their repository.

**Comment by arturovt** [comment]
Should be resolved this week and I'll be able to publish a minor version once Nx publishes new minor version too.

**Comment by arturovt** [comment]
Published 8.0.1.
